ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -201,6 +201,22 @@ RUN git clone https://github.com/kanaka/noVNC /home/user/.novnc/ \
  && cd /home/user/.novnc \
  && npm install \
  && node ./utils/use_require.js --as commonjs --with-app
+ 
+# Install dependencies to enable codeintel (https://github.com/c9/c9.ide.language.codeintel)
+RUN easy_install pip
+ && pip install -U pip
+ && pip install -U virtualenv
+ && virtualenv --python=python2 $HOME/.c9/python2
+ && source $HOME/.c9/python2/bin/activate
+ && apt-get update
+ && apt-get install python-dev
+ && mkdir /tmp/codeintel
+ && pip download -d /tmp/codeintel codeintel==0.9.3
+ && cd /tmp/codeintel
+ && tar xf CodeIntel-0.9.3.tar.gz
+ && mv CodeIntel-0.9.3/SilverCity CodeIntel-0.9.3/silvercity
+ && tar czf CodeIntel-0.9.3.tar.gz CodeIntel-0.9.3
+ && pip install -U --no-index --find-links=/tmp/codeintel codeintel
```

Comment:
Nit: Please delete the temporary `/tmp/codeintel` folder once you no longer need it. (And if some files need to be kept around for the installation to work, please move them somewhere under `/home/user`, e.g. as dotfiles or even better close to where it will be used, e.g. under a new folder in `/home/user/.c9` as we do with noVNC).

Reply:
They advocate to work under /tmp/ for this part, I assume pip does all the work, and nothing is necessary afterwards. done

Reply:
All these steps look like a very complicated way to do `pip install codeintel==0.9.3`. Wouldn't such a simple command work as well?